src/index.js

```diff
@@ -123,10 +157,12 @@ function startSearch(searchTerm) {
     .then(mvnSearchResponseArrived);
 }
 
-export function search(argv) {
-    if (argv.length < 3 || argv[2].trim() === "") {
-    newSearch();
+export function search(term, format) {
+    searchTerm = term;
+    dependencyFormat = format;
```

Comment:
I am sure yargs honer defaults, but to ensure it is not leaking `null` reference on code. Should we not do it something like this?
```js
searchTerm = term || ''; 
dependencyFormat = format || 'maven';
```

Reply:
done